### PR TITLE
RDM-3079: Search by case id not working with hyphens

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
RDM-3079: Search by case id not working with hyphens
    
           [RDM-3079] (https://tools.hmcts.net/jira/browse/RDM-3079).

### CODE REVIEW

File: elastic-search-support/src/main/resources/casesIndexSettings.json [hunk 1]
```diff
@@ -7,13 +7,25 @@
...
         "pattern":" ",
         "replacement":""
+      },
+      "hyphen_remove":{
+        "type":"pattern_replace",
+        "pattern":"-",
+        "replacement":""
       }
     }
   }
```
Comment (Thor-tech-of-metal) [inline elastic-search-support/src/main/resources/casesIndexSettings.json:31]: Make sure to add case_id_analyzer for testing folders and run test

Comment (Thor-tech-of-metal) [reply elastic-search-support/src/main/resources/casesIndexSettings.json:31]: DONE

**Review by kiran-yenigala-hmcts — APPROVED** [review]
LGTM

**Review by mario-paniccia — APPROVED** [review]
👍

**Pull request merged**